apps/modernization-ui/src/apps/page-builder/components/AddValueset/AddValueset.tsx

```diff
@@ -0,0 +1,161 @@
+                <input
+                    type="radio"
+                    name="isLocalOrPhin"
+                    value="phin"
+                    className="right-radio"
+                    checked={!isLocalOrPhin}
+                    onChange={(e: any) => setIsLocalOrPhin(!e.target.checked)}
+                />
+                <span className="radio-label">PHIN</span>
+                <br></br>
+                <div className={isValuesetNameNotValid ? 'error-border' : ''}>
+                    <label>
+                        Value set name<span className="mandatory-indicator">*</span>
+                    </label>
+                    <p className="error-text">Value set Name Not Valid</p>
+                    <input
+                        className="field-space"
+                        type="text"
+                        name="valuesetName"
+                        style={{ border: isValuesetNameNotValid ? '1px solid #dc3545' : '1px solid black' }}
+                        onBlur={(e: any) => validateValuesetName(e.target.value)}
+                        value={name}
+                        onInput={(e: any) => setName(e.target.value)}
+                    />
+                </div>
+                <div>
+                    <label>Value set description</label>
+                    <input
+                        className="field-space"
+                        type="text"
+                        name="valuesetDesc"
+                        style={{ border: '1px solid black' }}
+                        value={desc}
+                        onInput={(e: React.ChangeEvent<HTMLInputElement>) => setDesc(e.target.value)}
+                    />
+                </div>
+                <div className={isValuesetCodeNotValid ? 'error-border' : ''}>
+                    <label>
+                        Value set code<span className="mandatory-indicator">*</span>
+                    </label>
+                    <br></br>
+                    <p className="error-text">Value set code Not Valid</p>
+                    <input
+                        className="field-space"
+                        type="text"
+                        name="valuesetCode"
+                        style={{ border: isValuesetCodeNotValid ? '1px solid #dc3545' : '1px solid black' }}
+                        onBlur={(e: React.ChangeEvent<HTMLInputElement>) => validateValuesetCode(e.target.value)}
+                        value={code}
+                        onInput={(e: React.ChangeEvent<HTMLInputElement>) => setCode(e.target.value)}
+                    />
+                </div>
+            </div>
+            <ClassicButton className="submit-btn" url="" onClick={handleSubmit} disabled={isValidationFailure}>
+                Add to question
+            </ClassicButton>
+            <ClassicButton className="cancel-btn" url="" onClick={handleSubmit}>
+                Cancel
+            </ClassicButton>
```

Comment:
Classic buttons are intended for use in submitting requests to legacy NBS. Need to use standard buttons here.